aspnet/aspnet/overview/developing-apps-with-windows-azure/building-real-world-cloud-apps-with-windows-azure/continuous-integration-and-continuous-delivery.md

```diff
@@ -48,7 +48,7 @@ One of the things we're working on with Visual Studio Online (VSO), formerly kno
 - It supports team room collaboration, which facilitates real-time communication and collaboration for small agile teams.
 - It supports agile project management.
 
-For more information about the continuous integration and delivery features of VSO, see [Visual Studio Lab Management](https://www.visualstudio.com/en-us/explore/lab-management-vs.aspx) and [Visual Studio Release Management](https://www.visualstudio.com/en-us/explore/release-management-vs.aspx). An application monitoring feature, [Application Insights for Visual Studio Online](https://msdn.microsoft.com/en-us/library/dn481095.aspx), is in preview (available to try but not released for production use yet).
+For more information about the continuous integration and delivery features of VSTS, see [https://www.visualstudio.com/team-services/]
```

Comment:
* Link by doc name, which in this case is the VSTS webpage title
* Period at the end of the sentence
* I usually go with "on" over "about" when the object is the definitive source of information. The VSTS doc isn't just "about" the service ... it's **the doc "on" the service**.
```
For more information on the continuous integration and delivery features of VSTS, see [Visual Studio Team Services](https://www.visualstudio.com/team-services/).
```